### DISCUSSION TITLE
How to get one error currently occurred

### DISCUSSION THREAD

**Discussion by taejs** [react-hook-form/react-hook-form#7830] [Q&A]
Hi I wonder if I can get single error just occurred.
I tried to use a resolver option, but it's not support to default validation schema of react-hook-form.

**Comment by Moshyfawn** [comment]
Could you elaborate on what you're trying to achieve?

**Comment by taejs** [comment]
I use options 
```
useForm({
   mode : 'onSubmit',
   reValidateMode: 'onChange'
});
```

4 fieldNames are
firstname  /  lastname / phonenumber / hobby(selectbox)

all fields and use `rule: {required: true}` and I put my custom rule in phonenumber field that allowing only number

After I submitted this, errors look like this
```
{
   firstname: {type: 'required'},
   lastname: {type: 'required'},
   phonenumber: {type: 'integerOnly'},
   hobby: {type: 'required'},
}
```

In this situation, what I want to do is that when user change the phoneNumber field again and if it's not valid, I just want to get this error of phonenumber. Not all 4 errors.


@Moshyfawn

**Comment by taejs** [comment]
Or there's some way to get currently changing input?

**Comment by taejs** [comment]
https://github.com/react-hook-form/react-hook-form/issues/7561

I think it's similliar case what I want to do

**Comment by taejs** [comment]
the first field of errors is the last error? How about when it comes to Field Array?
